submissions/satisfactorio/satisfactorio.md

```diff
@@ -0,0 +1,91 @@
+
+### Extra features
+
+Satisfactory offers a remarkable number of features not present in Factorio, so these had to be replicated as closely as possible.
+
+Combat in the early-game is done with a melee weapon, and typically involves dodging enemy attacks while poking at them. Factorio doesn't really lend itself to this, but instead the player's weapon will deal knockback and stun. When used correctly, this allows you to defeat early-game enemies without even taking damage! The enemies themselves do not build bases, but instead they are found in limited number guarding points of interest such as resource nodes and collectable items.
+
+Not all creatures are hostile! In Satisfactory you can find and tame Lizard Doggos to keep as pets, and they will repay you with occasional random loot. This has been recreated in the mod, so you can have a little farm! Check in on them every so often to see what they've found. Maybe it'll be a Power Slug, or maybe it'll be nuclear waste...
+
+[[Video: doggo-farm.mp4 -- Collecting items from the farm]]
+
+Vehicles are equipped with an autopilot feature. Record a path and then set it running automatically. Great for early-game logistics before you get trains. Vehicles can be loaded and unloaded at Truck Stations, allowing for well organised and distributed factories.
+
+[[Video: self-driving.mp4 -- A car bringing in a delivery of encased industrial beams]]
+
+Need more movement options? Satisfactory provides the Zipline, which lets you travel along power lines. Hyper Tubes allow for travel between two connected locations. Later on you can make a Jetpack to fly a fair distance, and later still there is a Hover Pack that uses electricity from the power grid to fly as long as you want.
+
+[[Video: jetpack.mp4 -- Wheeeeeeeee!!!]]
+
+Drones allow for efficient long-distance transport of items. They're like Logistics Robots but actually good! Each drone can carry 9 stacks of items at once, waiting at the destination until they are all unloaded, before picking up any items that may be returned to the origin. They're blazing fast, reaching speeds over 240kmh! However, the loading and unloading process takes longer, making drones less effective for shorter journeys.
+
+[[Video: drone-port.mp4 -- This drone delivers packaged nitrogen gas, and takes empty canisters back to be refilled]]
+
+There's more, but this has already been a pretty long list!
+
+### Help wanted!
+
+As you have doubtless noticed from the screenshots above, I am a programmer, not a graphics person! Every building in the game is just a white card with the building's icon. It's perfectly playable in this state, but it doesn't look that good.
+
+I am looking for help with this. If you have any experience making graphics for Factorio mods, I'd love to hear from you and perhaps make something happen.
+
+In the meantime, please [enjoy the mod](https://mods.factorio.com/mod/Satisfactorio) - I've spent almost a year non-stop on it!
```

Comment:
I think you should mention here when it will be available? Are you planning on releasing it on the same day? Will it already be on the portal by the time the article goes live? Or only later?

Reply:
If I can coordinate with whoever pushes the button to make the article go live, then I can hit the Publish Mod button at the same time. Even then I should include "out now!" or *something*.

Reply:
I think it would be better if you publish it on the portal like an hour or whatever before the article goes live. Just so that you can test download it from the portal from a new install to see if everything works fine and whatever. But we can be in contact with you if you want it closer to the time :)

But yeah, regardless I agree you should add a "out now" or something somewhere :)